packages/main/poetry.lock

```diff
@@ -2162,27 +2162,6 @@ beautifulsoup4 = [
     {file = "beautifulsoup4-4.11.1.tar.gz", hash = "sha256:ad9aa55b65ef2808eb405f46cf74df7fcb7044d5cbc26487f96eb2ef2e436693"},
 ]
 black = [
-    {file = "black-22.3.0-cp310-cp310-macosx_10_9_universal2.whl", hash = "sha256:2497f9c2386572e28921fa8bec7be3e51de6801f7459dffd6e62492531c47e09"},
```

Comment:
No need to change this file since we're affecting **pdf** sub-package only.

Reply:
I believe this update may have been part of merging the master onto the current branch. My commit that I see in the commit history only shows the the pdf/poetry.lock file was updated.